crates/kitsune_p2p/kitsune_p2p/src/gossip/sharded_gossip.rs

```diff
@@ -296,6 +296,8 @@ impl ShardedGossipLocalState {
             } else {
                 self.metrics.record_success(state_key.clone());
             }
+        } else if init_tgt && error {
+            self.metrics.record_error(state_key.clone());
         }
```

Comment:
What is the significance of this else branch? We had an error in between deciding to initiate and starting the round?

Reply:
It's because if it's an initiate tgt then we might not have any round state yet so we still want to record the error when trying to initiate with nodes that are offline.